packages/next-server/server/config.ts

```diff
@@ -18,6 +18,7 @@ const defaultConfig: { [key: string]: any } = {
   pageExtensions: ['tsx', 'ts', 'jsx', 'js'],
   target: process.env.__NEXT_BUILDER_EXPERIMENTAL_TARGET || 'server',
   poweredByHeader: true,
+  crossOrigin: 'anonymous',
```

Comment:
This is needed to preload `type=module` scripts. We need to set a crossOrigin attribute in this case because[ `<script type="module">` with no crossorigin attribute indicates a credentials mode of `omit`](https://developers.google.com/web/updates/2017/12/modulepreload).

Reply:
Sure